front-end/src/components/main-page/TranscriptUploadModal.jsx

```diff
@@ -55,8 +55,10 @@ class TranscriptUploadModal extends React.Component {
       const parsedData = JSON.parse(data);
       if (parsedData.data){
         for (const intent of parsedData.data) {
-          if (intentIsMissingFields(intent)){
-            return false;
+          if (Object.keys(intent).length !== 0){
+            if (intentIsMissingFields(intent)){
+              return false;
```

Comment:
Would this not throw an error if we passed in an empty object?

Reply:
It wouldn't and that what we want isn't it? we want to use empty objects to differentiate between conversations in transcript